src/server.jl

```diff
@@ -327,39 +330,21 @@ function serve_file(
 end
 
 """
-    ws_upgrade(http::HTTP.Stream)
-
-Upgrade the HTTP request in the stream to a websocket.
-"""
-function ws_upgrade(http::HTTP.Stream)
```

Comment:
We don't really need this since we can just use the `HTTP.WebSockets.upgrade` function